src/JuliaFormatter.jl

```diff
@@ -549,6 +552,23 @@ function format_text(text::AbstractString, style::AbstractStyle; kwargs...)
     return format_text(text, style, opts)
 end
 
+function format_text(text::AbstractString, style::SciMLStyle; maxiters = 3, kwargs...)
+    isempty(text) && return text
+    opts = Options(; merge(options(style), kwargs)...)
+    # We need to iterate to a fixpoint because the result of short to long
+    # form isn't properly formatted
```

Comment:
maybe we should just make this part work properly then we wouldn't have to do this

Reply:
@YingboMa can you give a minimal example of this?

Reply:
It's just https://github.com/domluna/JuliaFormatter.jl/pull/588/files#diff-60c9ecaf817f34a35b04e83d428e16cd44a2bbf06b642a1fffc8f67d39e2aa9dR2-R12

Reply:
ok made an issue to track it, since this is function is only applicable to SciMLStyle this is fine for now.